### PR TITLE
Style for Dropdown

### PR DESCRIPTION
**Pull request by sanlucca**
- the icon for folded/unfolded will be replaced by an expand/collapse, it is only there for capture ;
- the component is (almost 100%) accessible;
- borders are not added if the folded part is at the top or bottom of the component;
- fixed width, but defined max height and max width;
- you can put buttons such as links.

Annotation 2020-05-19 155415

Annotation 2020-05-19 155414

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1002

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1002

File: packages/ng/libraries/root/src/style/components/_dropdown.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.lu-dropdown-options,
+.lu-dropdown-options-item-children {
+	border-radius: _theme("commons.border.radius", true);
+	border-radius: _theme("commons.border.radius");
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
+	width: 14rem; // magic number
```
Comment (RipripKipkip) [inline packages/ng/libraries/root/src/style/components/_dropdown.scss:13]: these comments should be removed ?

Comment (vvalentin-lucca) [reply packages/ng/libraries/root/src/style/components/_dropdown.scss:13]: I think it avoids the question of where the value comes from.

Comment (lucienbertin) [reply packages/ng/libraries/root/src/style/components/_dropdown.scss:13]: or, or, or, this should be a theme variable

Comment (vvalentin-lucca) [reply packages/ng/libraries/root/src/style/components/_dropdown.scss:13]: It's only about this component, so I don't know.

**Comment by lucienbertin** [comment]
this PR is incorporated in PR #1010, dont merge this one i'll handle it

**Pull request merged**